hw/ip_templates/rstmgr/dv/env/rstmgr_if.sv.tpl

```diff
@@ -3,6 +3,17 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 // clkmgr interface.
+<% 
```

Comment:
I think this can be even easier:
```
<% 
all_clks = set(clk_freqs.keys())

if "io_div4" in all_clks:
    preferred_domain = "io_div4"
elif "io" in all_clks:
    preferred_domain = "io"
else:
    assert 0, "No preferred clock available"

preferred_rst_n = f"rst_lc_{preferred_domain}_n"
%>\
```

Then the last line becomes just
```
  always_comb rst_ni_inactive = resets_o.${preferred_rst_n}[rstmgr_pkg::Domain0Sel];
```